src/build.d

```diff
@@ -215,6 +216,23 @@ will trigger a full rebuild.
 
 */
 
+/// Returns: The dependency that runs the autotester build
+alias autoTesterBuild = makeDep!((builder, dep) {
+    builder
+    .name("auto-tester-build")
+    .description("Runs the autotester build")
+    .deps([dmdDefault, checkwhitespace]);
+
+    version (Posix)
+        dep.deps ~= runCxxUnittest;
+
+    // unittests are currently not executed for auto-tester-test on windows and
+    // adding them requires more changes (see https://github.com/dlang/dmd/pull/7414).
+    // This requires no further actions and avoids building druntime+phobos on unittest failure
+    version (Windows)
+        dep.deps ~= runDmdUnittest;
```

Comment:
I'm not sure I follow the comment above. It says that unit tests don't work on Windows and yet we enable them only for Windows. What am I missing?

Reply:
I was referring to the approach of #7414 to add `ùnittest` to `auto-tester-test` in `./win32.mak`. This requires further work because the auto-tester applies patches to these makefiles which would need to adapted as well

Reply:
Clarified that this comment